### PR TITLE
Add [`cassette`/`video`]`-tape` icons

### PR DESCRIPTION
**Pull request by danielbayley**
Alts:
Artboard 1@2x

Untitled

### CODE REVIEW

**Comment by jguddas** [comment]
How about rotating the cassette 180º?

**Comment by danielbayley** [comment]
> How about rotating the cassette 180º?

@jguddas See updated comparison [above](https://github.com/lucide-icons/lucide/pull/1248#issue-1703689099)… Although I think cassette is more recognisable as it is—I’m pretty sure it usually shows up in it’s current orientation… Also helps distinguish it from VHS.

**Comment by jguddas** [comment]
I prefer the one in the second row second column.

Not sure about the videotape icon, it feels a bit cramped to me.

**Comment by danielbayley** [comment]
> I prefer the one in the second row second column.

Aiming to also be consistent with https://lucide.dev/icon/voicemail here…

File: icons/cassette-tape.svg [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <rect width="20" height="16" x="2" y="4" rx="2" />
+  <circle cx="8" cy="10" r="2" />
+  <path d="M8 12h8" />
+  <circle cx="16" cy="10" r="2" />
+  <path d="m6 20 .8-3c.1-.5.7-1 1.2-1h8c.6 0 1.1.4 1.2 1l.8 3" />
```
Comment (karsa-mistmere) [inline icons/cassette-tape.svg:16]: As usual, there are some issues with curvature, shame on you, Adobe Illustrator. :(
```suggestion
  <path d="m6 20 .7-2.9A1.4 1.4 0 0 1 8.1 16h7.8a1.4 1.4 0 0 1 1.4 1l.7 3"/>
```

**Review by karsa-mistmere — APPROVED** [review]
> > I prefer the one in the second row second column.
> 
> Aiming to also be consistent with https://lucide.dev/icon/voicemail here…

Yeah, it's also much less dense this way, this LGTM apart from the curvature issue!

**Comment by Unkn0wnCat** [comment]
Is it still planned to merge this? I personally would need these icons :p

**Comment by danielbayley** [comment]
> Is it still planned to merge this? I personally would need these icons :p

@Unkn0wnCat That’s up to @ericfennis

**Review by ericfennis — APPROVED** [review]
Yess sorry missed this one. Good to go!

**Pull request merged**